components/tutorial_tour_tip/constant.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// This needs to replaced with the actual tour data based on product
+
+export const FINISHED = 999;
+export const SKIPPED = -999;
```

Comment:
Rather than adding a new `SKIPPED`/`-999` status, it might be better for now to stop pushing the user to `999` when they successfully end a tour and instead increment their tour number by one so that they continue seeing new tour points when they come up.

A recent commit made it so users were always moved to `999` when they finished the trial. Before that commit, they only moved to `999` if they clicked to skip the tutorial. Otherwise, they were just incremented one step, and whenever the next step of tutorial was added, they saw that next step. Once at `999` status, they didn't see any more tutorial tips, even if some were added later.

Adding `SKIPPED`/`-999` in addition to pushing to FINISHED may be meaningful, but only if we're taking some action based on FINISHED vs SKIPPED status, which we currently aren't taking action on either status. I assume the action we'd take would be to move their status number to whatever the newly added tip is for FINISHED users but not for SKIPPED users (but we'd need to confirm that with product/UX). Coding that is fragile because it means tracking whether they've already seen that new step or not, which we can't do meaningfully without adding more persistent data somewhere.

I think a better approach would be to stop pushing to `FINISHED` when users finish a tutorial normally, not add a `SKIPPED` step, and migrate the disparate tour sets we are adding into user props once we finish the use case onboarding feature work.